pom.xml

```diff
@@ -169,7 +169,7 @@
                       <files>
                         <file>/etc/redhat-release</file>
                       </files>
-                      <content>release 6.7</content>
+                      <content>release 6.8</content>
```

Comment:
I think need also change message and comment above in this pom (RHEL 6.7 -> RHEL 6.8)?